test/performance/README.md

```diff
@@ -0,0 +1,25 @@
+# Profiles Performance Test
+
+The `profiles.jmx` [JMeter](http://jmeter.apache.org/) test will call the Profiles application for a number of different surgeries.
+Several parameters are exposed to customise the base test.
+
+
+| Parameter                         | Description                                                        | Default               |
+|:---------------------------------|:-------------------------------------------------------------------|-----------------------|
+| `hostname`                       | URL of server to test                                              | profiles.dev.beta.nhschoices.net|
+| `users`                          | Simulated number of concurrent users                               | 5                  |
+| `rampup`                         | Time in seconds to ramp up to total number of users                | 20                 |
+| `duration`                       | Time in seconds to run test                                        | 120                |
+| `throughput`                     | target throughput in samples per minute                            | 120                |
+| `csvfile`                        | csv data file of choices ID to use, either `ids_100.csv`, `ids_1200.csv`, or `ids_all.csv` (9000+), allowing a number of different GP pages to be visited during the test | ids_100.cs         |
+
+## Running the Test
+
+The test may be loaded into the JMeter application, or the file may be run from the command line locally by changing into the `test/performance` folder and entering:
+
+/usr/local/bin/jmeter/jmeter -n -t  ./profiles.jmx -Jhostname=*hostname* -Jusers=*users* -Jrampup=*rampup* -Jduration=*duration* -Jthroughput=*throughput* Jcsvfile=*csvfile* -l profiles.jtl
```

Comment:
You can either have italics from the *, or back ticks, they don't seem to play well together . I think italicising the variable names is better than the code formatting.

The scope of this work is to provide a JUnit test for the Teamcity performance checks. Providing a way to run performance tests locally is a big topic, as already mentioned in @beseven 's comment above. 

In my mind the only reason to run them locally is when editing the JMX file.